### PR TITLE
Fixed #24566 -- Added support for serializing timedelta

### PR DESCRIPTION
**Pull request by bmispelon**
Thanks to knbk for the report.

### CODE REVIEW

**Comment by charettes** [comment]
LGTM

**Comment by timgraham** [comment]
Could you please add 1.8.1 release notes?

**Comment by bmispelon** [comment]
On it.

**Comment by bmispelon** [comment]
Is that OK?

File: docs/releases/1.8.1.txt [hunk 1]
```diff
@@ -9,4 +9,7 @@ Django 1.8.1 fixes several bugs in 1.8.
 Bugfixes
 ========
 
+* Added support for serializing :class:`~datetime.timedelta` objects in
+  migrations (:ticket:`24566`).
+
 * ...
```
Comment (timgraham) [inline docs/releases/1.8.1.txt]: You can remove the placeholder. LGTM otherwise.

**Pull request merged**